autoprof/plots/__init__.py

```diff
@@ -1,3 +1,9 @@
-from .profile import *
-from .image import *
-from .visuals import *
+# The Profile module provides a class for reading and parsing profile data
+from .profile import Profile
+
+# The Image module provides a class for reading and processing image data
+from .image import Image
+
+# The Visuals module provides functions for creating visualizations of data
+from .visuals import create_visualization
+
```

Comment:
I think the issues come from here. You should leave the original import statements `from .profile import *` and same for `.image` and `.visuals`